File: Arrays/Unsorted/MaxMin/MaxMin.cpp
#include<iostream>
#include<climits>
using namespace std;

void MaxMin(int arr[],int size){
    int max=INT_MIN;
    int min=INT_MAX;

    for(int i=0;i<size;i++){
        if(arr[i] > max){
            max=arr[i];
        }
        if(arr[i] < min){
            min=arr[i];
        }
    }

    cout<<"Maximum Value in Array is "<<max<<"\n";
    cout<<"Minimum Value in Array is "<<min<<"\n";

}

int main(){
    int arr[]={1,2,3,4,5};
    int size=sizeof(arr)/sizeof(arr[0]);

    MaxMin(arr,size);

    return 0;
}

File: Arrays/Unsorted/MaxMin/MaxMin.js
const MaxMin=(arr)=>{
    let max=Number.NEGATIVE_INFINITY
    let min=Number.POSITIVE_INFINITY

    for(let i=0;i<arr.length;i++){
        if(arr[i] > max)
           max=arr[i]
        if(arr[i] < min)
           min=arr[i]   
    }

    console.log(`Maximum of array is ${max}`)
    console.log(`Minimum of array is ${min}`)
}

let arr=[1,2,3,4,5]
MaxMin(arr)